CONTRIBUTING.md

```diff
@@ -0,0 +1,16 @@
+# Contribution Guidelines
+If you are looking forward to contribute to this project, please read the following guidelines:
+
+### Set Up Project
+To set up and run the project on your system, please follow the steps mentioned [here](https://github.com/NJACKWinterOfCode/B.E.N.J.I./blob/master/README.md).
+
+### Raising an Issue
+
+If you look forward to contributing to this project as part of [NJACK Winter of Code](https://github.com/NJACKWinterOfCode), please report the issues [here](https://github.com/NJACKWinterOfCode/B.E.N.J.I./issues).
+
+
+### Sending Pull Request
+
+If you look forward to contributing to this project as part of [NJACK Winter of Code](), all pull requests should be made by creating a new branch of the [main repository](https://github.com/NJACKWinterOfCode/B.E.N.J.I.).

```

Comment:
Remove the NJACK winter of code stuff. In future use, it can be used elsewhere too. Add some more serious stuff like this guide [here](https://github.com/NJACKWinterOfCode/first-timers-guide/blob/master/CONTRIBUTING.md)